sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/parallel_for.py

```diff
@@ -108,6 +121,26 @@ def _create_instance_from_schema_dict(cls, pipeline_jobs, loaded_data):
             **loaded_data
         )
 
+    def _convert_output_meta(self, outputs):
+        """Convert output meta to aggregate types."""
+        # pylint: disable=protected-access
+        aggregate_outputs = {}
+        for name, output in outputs.items():
+            if output.type in self.OUT_TYPE_MAPPING:
+                new_type = self.OUT_TYPE_MAPPING[output.type]
+            else:
+                raise UserErrorException("Unsupported output type: {}.".format(output.type))
```

Comment:
Need to mention this is not supported in **parallel for** node?

Reply:
Good catch!